### ISSUE TITLE
Bug: Synchronize or monitor   clobbers the Access Control inheritance on folders in SharePoint

### ISSUE THREAD

**Issue by superspindr** [abraunegg/onedrive#2315]
### Describe the bug

I create a directory in my local SharePoint root directory (let me call it Testme).  I successfully do a --synchronize operation to the cloud.  I login to the Cloud and make sure that folder Testme  inherits the Access Controls for different user groups, the way I want.  Let me say Group-A is a 'readonly' group, who can't delete files and Group-B  are admin people who can do what they wish with the files. 

At first, all the individual files inside Testme folder als reflect these access control rules (click on ... -> Manage Access --> Advanced,  in SharePoint to see this).

Now, I add a few more files in my local instanceof Testme folder. I do a --synchronize again.  Once this operation is done, when I go and check the Cloud via a browser, all the files under Testme  have lost their inheritance rules from the parent.   
The folder 'Testme'   Access Control rules are still in tact. 

As a confirmation of this phenomenon,  when I access the SharePoint from a different computer as one of the users from the Group-A,   the folder Testme is there but I see this folder completely empty. 

The Attached file number : 1  shows the Access Rules for the parent directory I call "Amethyst" and 2 shows one of the files inside 'Amethyst', whose access control rules have been lost. 
![SharePoint_access_failure_1](https://user-images.githubusercontent.com/75458745/221320203-8937d476-e04f-4f6c-a951-b7ac8069e4a6.png)
![SharePoint_access_failure_2](https://user-images.githubusercontent.com/75458745/221320220-f9c3f79c-78c0-4a5f-b93b-880878bf9ab3.png)

### Operating System Details

```shell
Linux myhostname 3.10.0-1160.83.1.el7.x86_64 #1 SMP Wed Jan 25 16:41:43 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

CentOS Linux release 7.9.2009 (Core)
```

### Client Installation Method

From Source

### OneDrive Account Type

SharePoint

### What is your OneDrive Application Version

onedrive v2.5.0-dev

### What is your OneDrive Application Configuration

```shell
onedrive version                             = v2.5.0-dev
Config path                                  = /root/.config/onedrive
Config file found in config path             = false
Config option 'sync_dir'                     = /root/OneDrive
Config option 'enable_logging'               = false
Config option 'log_dir'                      = /var/log/onedrive/
Config option 'disable_notifications'        = false
Config option 'min_notify_changes'           = 5
Config option 'skip_dir'                     = 
Config option 'skip_dir_strict_match'        = false
Config option 'skip_file'                    = ~*|.~*|*.tmp
Config option 'skip_dotfiles'                = false
Config option 'skip_symlinks'                = false
Config option 'monitor_interval'             = 300
Config option 'monitor_log_frequency'        = 6
Config option 'monitor_fullscan_frequency'   = 12
Config option 'read_only_auth_scope'         = false
Config option 'dry_run'                      = false
Config option 'upload_only'                  = false
Config option 'download_only'                = false
Config option 'local_first'                  = false
Config option 'check_nosync'                 = false
Config option 'check_nomount'                = false
Config option 'resync'                       = false
Config option 'resync_auth'                  = false
Config option 'cleanup_local_files'          = false
Config option 'classify_as_big_delete'       = 1000
Config option 'disable_upload_validation'    = false
Config option 'bypass_data_preservation'     = false
Config option 'no_remote_delete'             = false
Config option 'remove_source_files'          = false
Config option 'sync_dir_permissions'         = 700
Config option 'sync_file_permissions'        = 600
Config option 'space_reservation'            = 52428800
Config option 'application_id'               = 
Config option 'azure_ad_endpoint'            = 
Config option 'azure_tenant_id'              = common
Config option 'user_agent'                   = 
Config option 'force_http_11'                = false
Config option 'debug_https'                  = false
Config option 'rate_limit'                   = 0
Config option 'operation_timeout'            = 3600
Config option 'sync_root_files'              = false
Selective sync 'sync_list' configured        = false
Config option 'sync_business_shared_folders' = false
Business Shared Folders configured           = false
Config option 'webhook_enabled'              = false
```

### What is your 'curl' version

```shell
curl 7.29.0 (x86_64-redhat-linux-gnu) libcurl/7.29.0 NSS/3.53.1 zlib/1.2.7 libidn/1.28 libssh2/1.8.0
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smtp smtps telnet tftp 
Features: AsynchDNS GSS-Negotiate IDN IPv6 Largefile NTLM NTLM_WB SSL libz unix-sockets
```

### Where is your 'sync_dir' located

Local

### What are all your system 'mount points'

```shell
sysfs on /sys type sysfs (rw,nosuid,nodev,noexec,relatime,seclabel)
proc on /proc type proc (rw,nosuid,nodev,noexec,relatime)
devtmpfs on /dev type devtmpfs (rw,nosuid,seclabel,size=7928332k,nr_inodes=1982083,mode=755)
securityfs on /sys/kernel/security type securityfs (rw,nosuid,nodev,noexec,relatime)
tmpfs on /dev/shm type tmpfs (rw,nosuid,nodev,seclabel)
devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,seclabel,gid=5,mode=620,ptmxmode=000)
tmpfs on /run type tmpfs (rw,nosuid,nodev,seclabel,mode=755)
tmpfs on /sys/fs/cgroup type tmpfs (ro,nosuid,nodev,noexec,seclabel,mode=755)
cgroup on /sys/fs/cgroup/systemd type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd)
pstore on /sys/fs/pstore type pstore (rw,nosuid,nodev,noexec,relatime)
efivarfs on /sys/firmware/efi/efivars type efivarfs (rw,nosuid,nodev,noexec,relatime)
cgroup on /sys/fs/cgroup/pids type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,pids)
cgroup on /sys/fs/cgroup/freezer type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,freezer)
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,net_prio,net_cls)
cgroup on /sys/fs/cgroup/perf_event type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,perf_event)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,cpuacct,cpu)
cgroup on /sys/fs/cgroup/blkio type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,blkio)
cgroup on /sys/fs/cgroup/devices type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,devices)
cgroup on /sys/fs/cgroup/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,cpuset)
cgroup on /sys/fs/cgroup/memory type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,memory)
cgroup on /sys/fs/cgroup/hugetlb type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,hugetlb)
configfs on /sys/kernel/config type configfs (rw,relatime)
/dev/nvme0n1p3 on / type ext4 (rw,relatime,seclabel,data=ordered)
selinuxfs on /sys/fs/selinux type selinuxfs (rw,relatime)
systemd-1 on /proc/sys/fs/binfmt_misc type autofs (rw,relatime,fd=36,pgrp=1,timeout=0,minproto=5,maxproto=5,direct,pipe_ino=10645)
debugfs on /sys/kernel/debug type debugfs (rw,relatime)
hugetlbfs on /dev/hugepages type hugetlbfs (rw,relatime,seclabel)
mqueue on /dev/mqueue type mqueue (rw,relatime,seclabel)
fusectl on /sys/fs/fuse/connections type fusectl (rw,relatime)
/dev/nvme0n1p2 on /boot type ext4 (rw,relatime,seclabel,data=ordered)
/dev/nvme0n1p1 on /boot/efi type vfat (rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=winnt,errors=remount-ro)
/dev/sda1 on /opt/nmrdata type ext4 (rw,relatime,seclabel,data=ordered)
sunrpc on /var/lib/nfs/rpc_pipefs type rpc_pipefs (rw,relatime)
tmpfs on /run/user/1000 type tmpfs (rw,nosuid,nodev,relatime,seclabel,size=1598244k,mode=700,uid=1000,gid=1000)
tmpfs on /run/user/42 type tmpfs (rw,nosuid,nodev,relatime,seclabel,size=1598244k,mode=700,uid=42,gid=42)
/dev/fuse on /run/user/1000/doc type fuse (rw,nosuid,nodev,relatime,user_id=1000,group_id=1000)
/dev/sda1 on /export/neo500 type ext4 (rw,relatime,seclabel,data=ordered)
nfsd on /proc/fs/nfsd type nfsd (rw,relatime)
tmpfs on /run/user/0 type tmpfs (rw,nosuid,nodev,relatime,seclabel,size=1598244k,mode=700)
```

### What are all your local file system partition types

```shell
NAME        FSTYPE LABEL UUID                                 MOUNTPOINT
sda                                                           
└─sda1      ext4         a783c459-3bf1-48ec-a448-16cc71d26ced /export/neo500
sr0                                                           
nvme0n1                                                       
├─nvme0n1p1 vfat         A39E-04BC                            /boot/efi
├─nvme0n1p2 ext4         17101a7d-e826-4954-80be-da5ee27a059a /boot
└─nvme0n1p3 ext4         079fd8e6-bbd7-483b-9330-7d9b9057a42a /
```

### How do you use 'onedrive'

I am trying to implement a central data storage and distribution of research data generated from analytical instruments in my department (Chemistry).  Our university (Univ. of Washington)  IT department provides enterprise level SharePoint access and provisioning for various labs and departments on Campus.    Under this paradigm, I have provisioned a SharePoint storage space for my department and the plan is to use onedrive Linux client from up to seven different Linux clients (attached to various instruments)  write raw data into the SharePoint space.  All our users (students, staff, faculty) have unique ID-s that get authenticated via a University wide LDAP backend.   They are grouped into diferent categories, such as 'nmr_users', 'nmr_admins'  and 'nmr_visitors', for example.    The University IT runs a dedicated Groups server with which we have created these groups and added different people from the cohort as members, based on our own criteria.  

The Groups server is also integrated to play with the SharePoint infrastructure so that, I can provide access control in a fine grained manner for data stored in the SharePoint space,  based on the above said group membership.  This is what I showed in the screenshots I attached above.    

### Steps to reproduce the behaviour

If you have a SharePoint Cloud space, you can try to replicate what I mentioned above. 

### Complete Verbose Log Output

```shell
Using 'user' Config Dir: /home/iconuser/.config/onedrive
Using 'system' Config Dir: /etc/onedrive
No user or system config file found, using application defaults
Checking Application Version ...
Initializing the OneDrive API ...
Configuring Global Azure AD Endpoints
Using Curl defaults for all HTTP operations
Opening the item database ...
All operations will be performed in: /home/iconuser/OneDrive
Application version: v2.5.0-dev
Account Type: business
Default Drive ID: b!8NNBgYmzRUm1rzWRul35EgmEyJ3JDdpMlEgomZfkYb_zjHOFRKH5T6cXO4x8rGqV
Default Root ID: 012SZBUQN6Y2GOVW7725BZO354PWSELRRZ
Remaining Free Space: 5493618066974
Fetching details for OneDrive Root
OneDrive Root exists in the database
Initializing the Synchronization Engine ...
Syncing changes and items from OneDrive ...
Applying changes of Path ID: 012SZBUQN6Y2GOVW7725BZO354PWSELRRZ
Updated Remaining Free Space: 5493618066974
Processing 201 OneDrive items to ensure consistent local state
Creating local directory: ./av500
Creating local directory: av500/rdempc
Creating local directory: av500/rdempc/nmr
Creating local directory: av500/rdempc/nmr/gencap3dimer31P2
Creating local directory: av500/rdempc/nmr/gencap3dimer31P2/1
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/cpdprg2 ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/acqus ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/acqu ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/scon2 ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/acqum ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/gencap3dimer31P2/1/pdata
Creating local directory: av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/peaklist.xml ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/peaks ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/format.temp ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/format.ased ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/fid ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P2/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/RD-2010-4 1H
Creating local directory: av500/rdempc/nmr/RD-2010-4 1H/1
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/acqus ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/acqu ... done.Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/scon2 ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/acqum ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/RD-2010-4 1H/1/pdata
Creating local directory: av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/format.temp ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/format.ased ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/fid ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 1H/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/protonstd
Creating local directory: av500/rdempc/nmr/protonstd/1
Downloading file av500/rdempc/nmr/protonstd/1/cpdprg2 ... done.
Downloading file av500/rdempc/nmr/protonstd/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/protonstd/1/acqus ... done.
Downloading file av500/rdempc/nmr/protonstd/1/acqu ... done.
Downloading file av500/rdempc/nmr/protonstd/1/spnam1 ... done.
Downloading file av500/rdempc/nmr/protonstd/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/protonstd/1/scon2 ... done.
Downloading file av500/rdempc/nmr/protonstd/1/acqum ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/protonstd/1/pdata
Creating local directory: av500/rdempc/nmr/protonstd/1/pdata/1
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/integrals.txt ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/peakrng ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/int1d ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/intrng ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/protonstd/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/protonstd/1/format.temp ... done.
Downloading file av500/rdempc/nmr/protonstd/1/format.ased ... done.
Downloading file av500/rdempc/nmr/protonstd/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/RD-2010-4 31P
Creating local directory: av500/rdempc/nmr/RD-2010-4 31P/1
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/acqus ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/acqu ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/scon2 ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/acqum ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pulseprogram ... done.
CrCreating local directory: av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/format.temp ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/format.ased ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/fid ... done.
Downloading file av500/rdempc/nmr/RD-2010-4 31P/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/2008-86-wash-Dz
Creating local directory: av500/rdempc/nmr/2008-86-wash-Dz/1
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/acqus ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/acqu ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/scon2 ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/acqum ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/2008-86-wash-Dz/1/pdata
Creating local directory: av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/format.temp ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/format.ased ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/fid ... done.
Downloading file av500/rdempc/nmr/2008-86-wash-Dz/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/gencap3dimer31P
Creating local directory: av500/rdempc/nmr/gencap3dimer31P/1
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/cpdprg2 ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/acqus ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/acqu ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/scon2 ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/acqum ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/gencap3dimer31P/1/pdata
Creating local directory: av500/rdempc/nmr/gencap3dimer31P/1/pdata/1
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/format.temp ... done.
--More--(37%)
eating local directory: av500/rdempc/nmr/RD-2010-4 31P/1/pdata
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/format.ased ... done.
Downloading file av500/rdempc/nmr/gencap3dimer31P/1/fid ... done.
Creating local directory: av500/rdempc/nmr/2008-83-Dz
Creating local directory: av500/rdempc/nmr/2008-83-Dz/1
Downloading file av500/rdempc/nmr/2008-83-Dz/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/acqus ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/acqu ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/scon2 ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/acqum ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/2008-83-Dz/1/pdata
Creating local directory: av500/rdempc/nmr/2008-83-Dz/1/pdata/1
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/format.temp ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/format.ased ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/fid ... done.
Downloading file av500/rdempc/nmr/2008-83-Dz/1/prosol_History ... done.
Creating local directory: av500/rdempc/nmr/2008-86-Dz
Creating local directory: av500/rdempc/nmr/2008-86-Dz/1
Downloading file av500/rdempc/nmr/2008-86-Dz/1/uxnmr.par ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/acqus ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/acqu ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/audita.txt ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/scon2 ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/acqum ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pulseprogram ... done.
Creating local directory: av500/rdempc/nmr/2008-86-Dz/1/pdata
Creating local directory: av500/rdempc/nmr/2008-86-Dz/1/pdata/1
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/1i ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/proc ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/procs ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/title ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/1r ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/outd ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/pdata/1/auditp.txt ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/format.temp ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/format.ased ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/fid ... done.
Downloading file av500/rdempc/nmr/2008-86-Dz/1/prosol_History ... done.
Creating local directory: av500/sdch
Creating local directory: av500/sdch/nmr
Creating local directory: av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2
Creating local directory: av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2/1
Downloading file av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2/1/uxnmr.par ... done.
Downloading file av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2/1/acqus ... done.
Downloading file av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2/1/acqu ... done.
Downloading file av500/sdch/nmr/Me3Si-phenyl-2phenyl-COOH dissolve in PhMe2/1/audita.txt ... done.
```

**Comment by abraunegg** [comment]
@superspindr 
Unfortunately this is not a bug with this client.

When you authenticate with this client, the OneDrive authentication scopes provides you with your security access into OneDrive - in your case SharePoint. Data that you upload with this client to SharePoint is allowed then through that access token.

Security inheritance of permissions from parental objects is something that the SharePoint Server should be doing and applying - as the OneDrive API has zero capability to configure | use | specify what security permissions to use / apply on the data being uploaded.

As such, this is not a bug with this client. Please raise this issue with Microsoft.

**Comment by abraunegg** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
